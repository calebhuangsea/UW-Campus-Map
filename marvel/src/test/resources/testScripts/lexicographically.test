# test finding Lexicographically least paths.

CreateGraph g

AddNode g n1
AddNode g n2

AddEdge g n1 n2 e1
AddEdge g n1 n2 a1

FindPath g n1 n2