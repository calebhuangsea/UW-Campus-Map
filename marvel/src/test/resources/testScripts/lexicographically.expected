# test finding Lexicographically least paths.

created graph g

added node n1 to g
added node n2 to g

added edge e1 from n1 to n2 in g
added edge a1 from n1 to n2 in g

path from n1 to n2:
n1 to n2 via a1